src/config/dark-sites.config

```diff
@@ -166,6 +167,7 @@ blocks.pandadev.net
 blog.aractus.com
 blog.counter-strike.net
 blog.ja-ke.tech
+blog.jothin.tech
```

Comment:
Site has theme switch so it's disqualified from Dark Sites.
![obraz](https://github.com/user-attachments/assets/78980129-6fb0-41d2-94b4-7c9648b0b444)